src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -15,14 +18,17 @@ public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose attributes contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
```

Comment:
Would it be better to say something along the line of "Finds all persons whose attributes contain the keyword (case-insensitive) specified for those attributes..." as it currently sounds a little like "if any attribute contains the keyword" or many keywords can be used for each attribute

Reply:
I agree, updated the message.